src/test/java/ru/vk/itmo/CompactionTest.java

```diff
@@ -147,4 +150,112 @@ void compactAndAdd(Dao<String, Entry<String>> dao) throws IOException {
         assertSame(dao.all(), entries);
     }
 
+    @DaoTest(stage = 4)
```

Comment:
Давайте следующим этапом, чтобы не ломать успешные решения:
```suggestion
    @DaoTest(stage = 5)
```